go.mod

```diff
@@ -52,6 +52,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/1Password/connect-sdk-go v1.2.0
```

Comment:
Sorry, one small thing. I would like this to be up in the above require block. Seems a little odd to have another require block with just one package.

Reply:
Sure. `go mod tidy` put this here. Not sure why.